### DISCUSSION TITLE
Weird behaviour

### DISCUSSION THREAD

**Discussion by maticboncina** [aviolaris/instaunfollowers#66] [Q&A] [answered]
Hello,

So I set this up as described in the docs and looking at it, it's not correct.

For example, there is "Apple" @apple in the list, which I am 100% sure never followed me, so they could have never unfollowed me either. 

Does this combine my unfollow and the people who unfollow me?

Thank you. 

Best, Matic

**Comment by aviolaris** [comment, marked as answer]
Hello!

Thank you for reaching out and expressing your concern.

I understand that, in a literal sense, an "unfollower" is someone who has stopped following another user on social media. However, in the context of the current project (and many other projects), the term "unfollowers" refers to anyone you are following but is not following you at the time of use. The information you are seeking cannot be extracted from the **Instagram Information File** because it does not contain that specific data. Given that, the behavior you described is the expected one, showing that you follow Apple, but Apple doesn't follow you back.

In the not-so-distant future, there might be a release that will store a snapshot of the followers in a database to enable tracking of those who may later unfollow. Therefore, you can click the `Watch` button, opting to be notified of new releases if the current one does not meet your needs.

**Reply by maticboncina** [reply]
Thank you for the reply. Makes sense and very good spot which didn't even come across my mind that a database would be needed for what I wanted :)
